src/scenes/Leaderboard/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import { inject, observer } from 'mobx-react';
+import { injectIntl, defineMessages } from 'react-intl';
+import { Grid, withStyles, Card as _Card, CardContent, Typography } from '@material-ui/core';
+
+import styles from './styles';
+import _Leaderboard from '../Event/components/Leaderboard';
+import { Row, Content } from '../Event/components';
+import { satoshiToDecimal } from '../../helpers/utility';
+
+const messages = defineMessages({
+  totalEvents: {
+    id: 'leaderboard.totalEvents',
+    defaultMessage: 'Total Events',
+  },
+  totalParticipants: {
+    id: 'leaderboard.totalParticipants',
+    defaultMessage: 'Total Participants',
+  },
+  totalQTUM: {
+    id: 'leaderboard.totalQTUM',
+    defaultMessage: 'Total bet in QTUM',
+  },
+  totalBOT: {
+    id: 'leaderboard.totalBOT',
+    defaultMessage: 'Total voted in BOT',
+  },
+});
+
+@withStyles(styles, { withTheme: true })
+@injectIntl
+@inject('store')
+@observer
+export default class Leaderboard extends Component {
+  componentDidMount() {
+    this.props.store.leaderboard.init();
+  }
+
+  render() {
+    const { eventCount, participantsCount, totalBOT, totalQTUM } = this.props.store.leaderboard;
+    return (
+      <Row>
+        <SidebarContainer>
+          <Card title={messages.totalEvents} value={eventCount} />
+          <Card title={messages.totalParticipants} value={participantsCount} />
+          <Card title={messages.totalQTUM} value={satoshiToDecimal(totalQTUM)} />
+          <Card title={messages.totalBOT} value={satoshiToDecimal(totalBOT)} />
```

Comment:
I meant to change this at the model level, but since not going the model route, this is better suited to be in the store. UI components should just get data and display, no manipulation.